### ISSUE TITLE
Can't execute image launch script: Line too long

### ISSUE THREAD

**Issue by Tonne-TM** [beryx/badass-runtime-plugin#27]
I have a kind of funny bug.

- Windows 10
- tried in cmd / powershell 5 / powershell core 6.1

After creating the image i try to start the launchscript:
```gradle
gradle runtime
...
.\build\image\bin\launcher
``` 

Then the following error appears:
```
Die eingegebene Zeile ist zu lang.
Syntaxfehler. 
```
Which means that a line is to long in the launchscript.

I think this is the problematic line in `.\build\image\bin\launcher`:
```
REM set CLASSPATH=%APP_HOME%\lib\landingpage-0.0.1-SNAPSHOT.jar;%APP_HOME%\lib\shared-0.0.1-SNAPSHOT.jar;%APP_HOME%\lib\spring-boot-starter-web-2.1.7.RELEASE.jar;%APP_HOME%\lib\spring-boot-starter-thymeleaf-2.1.7.RELEASE.jar;%APP_HOME%\lib\braintree-java-2.89.0.jar;%APP_HOME%\lib\lombok-1.18.6.jar;%APP_HOME%\lib\hazelcast-spring-3.11.4.jar;%APP_HOME%\lib\hazelcast-3.11.4.jar;%APP_HOME%\lib\spring-boot-starter-data-mongodb-reactive-2.1.7.RELEASE.jar;%APP_HOME%\lib\spring-boot-starter-json-2.1.7.RELEASE.jar;%APP_HOME%\lib\spring-boot-starter-2.1.7.RELEASE.jar;%APP_HOME%\lib\spring-boot-starter-logging-2.1.7.RELEASE.jar;%APP_HOME%\lib\netty-all-4.1.38.Final.jar;%APP_HOME%\lib\iextrading4j-all-3.1.1.jar;%APP_HOME%\lib\iextrading4j-client-3.1.1.jar;%APP_HOME%\lib\iextrading4j-api-3.1.1.jar;%APP_HOME%\lib\jackson-datatype-jdk8-2.9.9.jar;%APP_HOME%\lib\jackson-datatype-jsr310-2.9.9.jar;%APP_HOME%\lib\jackson-module-parameter-names-2.9.9.jar;%APP_HOME%\lib\jersey-media-json-jackson-2.27.jar;%APP_HOME%\lib\jackson-module-jaxb-annotations-2.9.9.jar;%APP_HOME%\lib\jackson-databind-2.9.9.jar;%APP_HOME%\lib\jackson-annotations-2.9.0.jar;%APP_HOME%\lib\ta4j-core-0.12.jar;%APP_HOME%\lib\spring-boot-starter-tomcat-2.1.7.RELEASE.jar;%APP_HOME%\lib\hibernate-validator-6.0.17.Final.jar;%APP_HOME%\lib\spring-webmvc-5.1.9.RELEASE.jar;%APP_HOME%\lib\spring-web-5.1.9.RELEASE.jar;%APP_HOME%\lib\thymeleaf-spring5-3.0.11.RELEASE.jar;%APP_HOME%\lib\thymeleaf-extras-java8time-3.0.4.RELEASE.jar;%APP_HOME%\lib\org.osgi.core-4.2.0.jar;%APP_HOME%\lib\jackson-jr-objects-2.9.9.jar;%APP_HOME%\lib\commons-csv-1.4.jar;%APP_HOME%\lib\spring-boot-autoconfigure-2.1.7.RELEASE.jar;%APP_HOME%\lib\spring-boot-2.1.7.RELEASE.jar;%APP_HOME%\lib\jersey-client-2.27.jar;%APP_HOME%\lib\jersey-hk2-2.27.jar;%APP_HOME%\lib\jersey-common-2.27.jar;%APP_HOME%\lib\hk2-locator-2.5.0-b42.jar;%APP_HOME%\lib\hk2-api-2.5.0-b42.jar;%APP_HOME%\lib\hk2-utils-2.5.0-b42.jar;%APP_HOME%\lib\javax.annotation-api-1.3.2.jar;%APP_HOME%\lib\spring-data-mongodb-2.1.10.RELEASE.jar;%APP_HOME%\lib\spring-context-5.1.9.RELEASE.jar;%APP_HOME%\lib\spring-aop-5.1.9.RELEASE.jar;%APP_HOME%\lib\spring-tx-5.1.9.RELEASE.jar;%APP_HOME%\lib\spring-data-commons-2.1.10.RELEASE.jar;%APP_HOME%\lib\spring-beans-5.1.9.RELEASE.jar;%APP_HOME%\lib\spring-expression-5.1.9.RELEASE.jar;%APP_HOME%\lib\spring-core-5.1.9.RELEASE.jar;%APP_HOME%\lib\snakeyaml-1.23.jar;%APP_HOME%\lib\logback-classic-1.2.3.jar;%APP_HOME%\lib\log4j-to-slf4j-2.11.2.jar;%APP_HOME%\lib\jul-to-slf4j-1.7.26.jar;%APP_HOME%\lib\mongodb-driver-3.8.2.jar;%APP_HOME%\lib\mongodb-driver-reactivestreams-1.9.2.jar;%APP_HOME%\lib\mongodb-driver-async-3.8.2.jar;%APP_HOME%\lib\reactor-core-3.2.11.RELEASE.jar;%APP_HOME%\lib\thymeleaf-3.0.11.RELEASE.jar;%APP_HOME%\lib\slf4j-api-1.7.26.jar;%APP_HOME%\lib\tomcat-embed-websocket-9.0.22.jar;%APP_HOME%\lib\tomcat-embed-core-9.0.22.jar;%APP_HOME%\lib\tomcat-embed-el-9.0.22.jar;%APP_HOME%\lib\validation-api-2.0.1.Final.jar;%APP_HOME%\lib\jboss-logging-3.3.2.Final.jar;%APP_HOME%\lib\classmate-1.4.0.jar;%APP_HOME%\lib\jackson-core-2.9.9.jar;%APP_HOME%\lib\spring-jcl-5.1.9.RELEASE.jar;%APP_HOME%\lib\logback-core-1.2.3.jar;%APP_HOME%\lib\log4j-api-2.11.2.jar;%APP_HOME%\lib\mongodb-driver-core-3.8.2.jar;%APP_HOME%\lib\bson-3.8.2.jar;%APP_HOME%\lib\reactive-streams-1.0.2.jar;%APP_HOME%\lib\guava-27.1-jre.jar;%APP_HOME%\lib\jersey-entity-filtering-2.27.jar;%APP_HOME%\lib\javax.ws.rs-api-2.1.1.jar;%APP_HOME%\lib\socket.io-client-1.0.0.jar;%APP_HOME%\lib\jaxb-runtime-2.3.1.jar;%APP_HOME%\lib\jaxb-api-2.3.1.jar;%APP_HOME%\lib\jaxb-impl-2.3.1.jar;%APP_HOME%\lib\activation-1.1.1.jar;%APP_HOME%\lib\attoparser-2.0.5.RELEASE.jar;%APP_HOME%\lib\unbescape-1.1.6.RELEASE.jar;%APP_HOME%\lib\failureaccess-1.0.1.jar;%APP_HOME%\lib\listenablefuture-9999.0-empty-to-avoid-conflict-with-guava.jar;%APP_HOME%\lib\jsr305-3.0.2.jar;%APP_HOME%\lib\checker-qual-2.5.2.jar;%APP_HOME%\lib\error_prone_annotations-2.2.0.jar;%APP_HOME%\lib\j2objc-annotations-1.1.jar;%APP_HOME%\lib\animal-sniffer-annotations-1.17.jar;%APP_HOME%\lib\javax.inject-2.5.0-b42.jar;%APP_HOME%\lib\engine.io-client-1.0.0.jar;%APP_HOME%\lib\json-20090211.jar;%APP_HOME%\lib\javax.activation-api-1.2.0.jar;%APP_HOME%\lib\txw2-2.3.1.jar;%APP_HOME%\lib\istack-commons-runtime-3.0.7.jar;%APP_HOME%\lib\stax-ex-1.8.jar;%APP_HOME%\lib\FastInfoset-1.2.15.jar;%APP_HOME%\lib\osgi-resource-locator-1.0.1.jar;%APP_HOME%\lib\aopalliance-repackaged-2.5.0-b42.jar;%APP_HOME%\lib\javassist-3.22.0-CR2.jar;%APP_HOME%\lib\okhttp-3.8.1.jar;%APP_HOME%\lib\javax.inject-1.jar;%APP_HOME%\lib\okio-1.13.0.jar

```

Is this a bug of the plugin or the openJDK or just  a stupid windows os problem?

**Comment by siordache** [comment]
Fixed in 1.5.1. Give it a try!

**Comment by Tonne-TM** [comment]
Thanks it is working now!
